lib/store.js

```diff
@@ -60,6 +74,7 @@ export function get(path, defaultValue) {
 // Utility
 
 function setIn(o, path, val) {
+  console.log("setin", path, 1, val, 2);
```

Comment:
I added that line for debugging during our pair-programming, - should be removed again ;)